docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -398,6 +399,12 @@ The same path should be used for `MISO_FILES_DIR` as is set for `miso.fileStorag
     unzip -xjo $CATALINA_HOME/webapps/ROOT.war 'WEB-INF/lib/sqlstore-*.jar' -d lib
     ./flyway -user=$MISO_DB_USER -password=$MISO_DB_PASS -url=$MISO_DB_URL -outOfOrder=true -locations=classpath:db/migration,classpath:uk.ac.bbsrc.tgac.miso.db.migration migrate -placeholders.filesDir=${MISO_FILES_DIR}
 
+`$DB_URL` should be in the same format as in the `ROOT.xml`, except replacing `&amp;` with just `&`:
+
+```
+jdbc:mysql://localhost:3306/lims?autoReconnect=true&zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF-8
```

Comment:
Should we advise to strip the options off the URL? Mine ends at 'lims' and works just fine

Reply:
We've had all kinds of weird Unicode problems without the encoding.

Reply:
We've been using these options in production and testing for a long time. They may or may not be necessary, but we know they're not harmful, so I'd rather keep them for now.